.github/workflows/nodejs.yml

```diff
@@ -29,6 +29,6 @@ jobs:
       test-command: npm run coverage:ci
       post-test-steps: |
         - name: Upload coverage report to Codecov
-          uses: codecov/codecov-action@v3
-    strategy:
-      fail-fast: false
+          uses: codecov/codecov-action@v4
+          with:
+            token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Looks like secrets are not allowed here: https://github.com/nodejs/node-core-utils/actions/runs/8170900413
I'll remove the codecov update from this PR.